src/main/java/edu/oregonstate/cs361/battleship/Main.java

```diff
@@ -1,21 +1,26 @@
-package edu.oregonstate.cs361.battleship;
```

Comment:
Won't changing the package name in Mian.java cause problems in the rest of the code?